.rubocop_todo.yml

```diff
@@ -392,27 +392,12 @@ Naming/VariableNumber:
 Performance/Caller:
   Enabled: false
 
-# Offense count: 9
-# Cop supports --auto-correct.
-Performance/Casecmp:
-  Enabled: false
```

Comment:
I don't like this one too much, is there a chance to keep it disabled? How big performance boost it brings?

Reply:
It depends on the length of the string, some quick benchmarks indicate about 1.2x faster for 4 letter string and about 2.2x faster for comparing a 50 letter string to a 4 letter string. Considering afaict none of these occurrences are in a very critical path that gets called very rapidly I'd say we can ignore this cop. I'll revert these changes.

Reply:
fixed.